packages/sitecore-jss-nextjs/src/components/NextImage.tsx

```diff
@@ -48,6 +49,11 @@ export const NextImage: React.FC<NextImageProps> = withFieldMetadata<NextImagePr
       if (!img) {
         return null;
       }
+      const {
+        sitecoreContext: { pageState },
+      } = useSitecoreContext();
+      // disable image optimization for Edit and Preview
+      const unoptimized = pageState !== LayoutServicePageState.Normal;
```

Comment:
In normal mode, we need to consider the customer may have specified `unopotimized={false}` (i.e. in `otherProps` here). We should use what they've specified in that case.

Reply:
It would be the other way around actually. unoptimized will be false in Normal mode by default. It is when customer set it to `true` we need to preserve the value. Adjusted for this case.

Reply:
Ah, yep correct